pkg/cli/alpha.go

```diff
@@ -47,6 +48,7 @@ Alpha subcommands are for unstable features.
 	for i := range alphaCommands {
 		alpha.AddCommand(alphaCommands[i])
 	}
+	alpha.AddCommand(c.newScaffoldCmd())
```

Comment:
Any new alpha subcommand can be defined here; https://github.com/kubernetes-sigs/kubebuilder/blob/e7ee3941b34e3ab56362dfaa6f945bfb15a75f8b/pkg/cli/alpha.go#L32-L34

Could you please change it out?

Reply:
If we want to reuse the functions that init, create, edit, etc us, we cannot put the command in the alpha package use `CLI`. If we do the latter, I have to add the command here. 

Thoughts on how to handle init, create, edit? The `CLI` struct has a lot of the logic already.

Reply:
If we want to reuse the functions that init, create, edit, etc us, we cannot put the command in the alpha package and instead use the `cli` package. If use the latter package, I wouldn't be able to add here given I need an instance of `CLI`. 

Thoughts on how to proceed with reusing the logic in init, create, and edit?

Reply:
The code here will add as alpha any new command added to that list.
Because of that we just set it in the list. 


> Thoughts on how to proceed with reusing the logic in init, create, and edit?

We can indeed exec the kubebuilder commands as we do in the e2e tests.
That will be like `cmd = exec.Command("kubebuilder", "init")` for example